lib/collection/proxy-config.js

```diff
@@ -3,21 +3,25 @@ var _ = require('../util').lodash,
     nodeUrl = require('url'),
     Url = require('./url').Url,
     UrlMatchPattern = require('../url-pattern/url-match-pattern').UrlMatchPattern,
+    PROTOCOL_DELIMITER = UrlMatchPattern.PROTOCOL_DELIMITER,
     ProxyConfig,
     E = '',
     COLON = ':',
     DEFAULT_PORT = 8080,
     PROTOCOL_HOST_SEPARATOR = '://',
+    MATCH_ALL_HOST_AND_PATH = '*/*',
     DEFAULT_PROTOCOL = 'http',
-    AllOWED_PROTOCOLS = ['http', 'https'];
+    AllOWED_PROTOCOLS = ['http', 'https'],
+    // 'http+https://*/*'
+    DEFAULT_PATTERN = AllOWED_PROTOCOLS.join(PROTOCOL_DELIMITER) + PROTOCOL_HOST_SEPARATOR + MATCH_ALL_HOST_AND_PATH;
 
 /**
 * The following is the object structure accepted as constructor parameter while calling `new ProxyConfig(...)`. It is
 * also the structure exported when {@link Property#toJSON} or {@link Property#toObjectResolved} is called on a
 * Proxy instance.
 * @typedef ProxyConfig~definition
 *
-* @property {String=} [match = '<all_url>'] The match for which the proxy needs to be configured.
+* @property {String=} [match = 'http+https://*\/*'] The match for which the proxy needs to be configured.
```

Comment:
please note this will literally publish `http+https://*\/*` (removing slash will break jsDoc)
couldn't find a way to publish `http+https://*/*`
Already a issue open here https://github.com/jsdoc3/jsdoc/issues/821